pulumi/grapl/__main__.py

```diff
@@ -181,16 +182,16 @@ def main() -> None:
     rust_log_levels = ",".join(
         [
             "DEBUG",
-            "serde_xml_rs=WARN",
+            "h2::codec=WARN",
             "hyper=WARN",
             "rusoto_core=WARN",
             "rustls=WARN",
+            "serde_xml_rs=WARN",
         ]
     )
     py_log_level = "DEBUG"
 
-    # We've seen some potentially false failures from the default 5m timeout.
-    nomad_grapl_core_timeout = "8m"
+    nomad_grapl_core_timeout = "5m"
```

Comment:
Did we sort out what was causing this?

Reply:
_probably_ it was Plugin Registry never successfully coming up, because it didn't have a real postgres instance to connect to - see 812

Now that artifacts work we'll be able to further debug this stuff if it happens again in the future.

In the meantime, this does work on my sandbox.